src/components/Kv/KvBaseInput.vue

```diff
@@ -1,42 +1,66 @@
 <template>
-	<label class="kv-base-input" :for="name">
-		<slot></slot>
-		<div class="kv-base-input__wrapper">
-			<input
+	<div>
+		<template v-if="type === 'checkbox'">
+			<kv-checkbox
 				:id="name"
 				:name="name"
-				class="kv-base-input__input"
-				:class="{ 'kv-base-input__input--error': validation.$error }"
+				:valid="!validation.$error"
```

Comment:
This will make the checkbox border red after https://github.com/kiva/kv-ui-elements/pull/134 is merged